extensions/odoo_theme/layout_templates/homepage.html

```diff
@@ -42,13 +42,13 @@ <h5 class="text-muted pt-3 text-uppercase fw_semibold">{{ _("Top Links") }}</h5>
                         </a>
                     </li>
                     <li>
-                        <a href="{{ pathto('administration/deployment/deploy') }}" class="stretched-link">
-                            {{ _("System configuration") }}
+                        <a href="{{ pathto('administration/maintain/update') }}" class="stretched-link">
+                            {{ _("Bugfix updates") }}
                         </a>
                     </li>
                     <li>
-                        <a href="{{ pathto('administration/maintain/update') }}" class="stretched-link">
-                            {{ _("Bugfix updates") }}
+                        <a href="{{ pathto('administration/upgrade') }}" class="stretched-link">
+                            {{ _("Upgrading Odoo") }}
```

Comment:
Okay why not indeed. I agree that upgrading may be more interesting than deploying ^^

Reply:
The idea is to bring back the "maintain" section on the homepage when it is reworked to have a bit more structure and a general intro.